### PR TITLE
remove tty in docker run

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ MAKEFLAGS += -B
 
 DOCKER_RUN := docker run --rm=true --privileged \
     -v $(PWD):/sonic \
-    -it sonic-slave-$(USER)
+    sonic-slave-$(USER)
```
Comment (qiluo-msft) [inline Makefile]: Maybe not necessary in this case, I think we can keep '-i'. It is possible that some internal binary will detect environment and run conditional code.

Comment (marian-pritsak) [inline Makefile]: Agree with @qiluo-msft 
I'd add something like -i$(SONIC_SLAVE_TTY)

Comment (lguohan) [inline Makefile]: what is SONIC_SLAVE_TTY?

Comment (qiluo-msft) [inline Makefile]: @marian-pritsak What is 'SONIC_SLAVE_TTY'? If not used, we can remove it.

**Pull request merged**